content/applications/finance/expenses/expense_reports.rst

```diff
@@ -0,0 +1,109 @@
+If needed, update the :guilabel:`Journal` field, using the drop-down menu.
+
+.. image:: expense_reports/expense-report-summary.png
+   :align: center
+   :alt: Enter a short description and select a manager for the report.
+
+If some expenses are missing from the report, they can still be added. Click :guilabel:`Add a line`
+at the bottom of the :guilabel:`Expense` tab. A pop-up window appears with all the available
+expenses that can be added to the report, with a status of :guilabel:`To Submit`. Tick the checkbox
+next to each expense being added, then click :guilabel:`Select`.
+
+The items now appear on the report that was just created. If a new expense needs to be added that
+does **not** appear on the list, click :guilabel:`New` to :ref:`create a new expense <expenses/new>`
+and add it to the report.
+
+.. image:: expense_reports/add-an-expense-line.png
+   :align: center
+   :alt: Add more expenses to the report before submitting.
+
+.. note::
+   Expense reports can be created in one of three places:
+
+   #. Navigate to the main :menuselection:`Expenses` app dashboard (also accessed by going to
+      :menuselection:`Expenses app --> My Expenses --> My Expenses`)
+   #. Navigate to :menuselection:`Expenses app --> My Expenses --> My Reports`
+   #. Navigate to :menuselection:`Expenses app --> Expense Reports`
+
+   In any of these views, click :guilabel:`New` to create a new expense report.
+
+.. _expenses/submit:
+
+Submit an expense report
+========================
+
+When an expense report is completed, the next step is to submit the report to a manager for
+approval. To view all expense reports, navigate to :menuselection:`Expenses app --> My Expenses -->
+My Reports`. Open the specific report from the list of expense reports.
+
+.. note::
+   Reports must be individually submitted, and **cannot** be submitted in batches.
+
+If the list is large, grouping the results by status may be helpful since only reports that have a
+:guilabel:`To Submit` status need to be submitted; reports with an :guilabel:`Approved` or
+:guilabel:`Submitted` status do not.
+
+The :guilabel:`To Submit` expenses are identifiable by the :guilabel:`To Submit` status, and by the
+blue text, while the other expenses text appears in black.
+
+.. image:: expense_reports/expense-status.png
+   :align: center
+   :alt: Submit the report to the manager.
+
+.. note::
+   The status of each report is shown in the :guilabel:`Status` column on the right. If the
+   :guilabel:`Status` column is not visible, click the :icon:`oi-settings-adjust`
+   :guilabel:`(Additional Options)` icon at the end of the row, and enable :guilabel:`Status`.
+
+Click on a report to open it, then click :guilabel:`Submit To Manager`. After submitting a report,
+the next step is to wait for the manager to approve it.
+
+.. important::
+   The :ref:`expenses/approve`, :ref:`expenses/post`, and :ref:`expenses/reimburse` sections are
+   **only** for users with the *necessary rights*.
```

Comment:
```suggestion
   **only** for users with the appropriate *Access Rights*.
```
SUGGESTION, as I'm not sure the 'necessary rights' element of the sentence requires italics, since it is not a specific field/functionality found on a different portion of the UI (not on screen at the moment)